src/components/backend-ai-resource-monitor.ts

```diff
@@ -752,7 +753,7 @@ export default class BackendAiResourceMonitor extends BackendAIPage {
       </div>
       ` : html``}
       <div class="layout ${this.direction}-card flex wrap">
-        <div id="resource-gauges" class="layout ${this.direction} ${this.direction}-panel resources flex wrap">
+        <div id="resource-gauges" class="layout horizontal ${this.direction}-panel resources flex wrap">
```

Comment:
It should be kept to original implementation. If you want to show the panel on another view, it is better to change the CSS part, not the class casting.

Reply:
I accepted the feedback and modified it to leave the original implementation as it is. Please review it again 😃
@inureyes